qa/workunits/fs/misc/osd_full_mgr_hang.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh -x
+
+set -e
+
+ceph fs subvolume create cephfs sub_0
+subvol_path=$(ceph fs subvolume getpath cephfs sub_0 2>dev/null)
```

Comment:
redirect stedrr to /dev/null and not dev/null

Reply:
done